pages/pivappintmodels.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: default
+title: PIV Application Integration Models
+permalink: /pivappintmodels/
```

Comment:
Thanks @dasgituser 

Can you move this markdown under the _devconfig?
And do a pull request to the _staging_ branch instead of _gh-pages_ branch?
We're working on testing the integrations with Federalist (for hosting) and gh-pages will be removed during the deployment

Appreciate the translation to markdown.